source/guides/technology-documentation-sites/index.html.md.erb

```diff
@@ -61,14 +61,14 @@ All the workspace-related settings (workspace ID, API key and API secret) can be
 
 Next we must setup the GitHub Actions to publish:
 
-3. Copy the automation script from [here](https://github.com/DFE-Digital/academy-transfers-technical-documentation/blob/master/source-c4-diagrams/export-private-cloud-diagrams.js) to `<repository-root>/source-c4-diagrams/export-private-cloud-diagrams.js`. 
+3. Copy the automation script from [here](https://github.com/DFE-Digital/academy-transfers-technical-documentation/blob/master/source-c4-diagrams/export-private-cloud-diagrams.js) to `<repository-root>/source-c4-diagrams/export-private-cloud-diagrams.js`.
```

Comment:
Can we make the link more descriptive here? It's not great for people using screen readers. See [the guidance on GOV.UK](https://www.gov.uk/guidance/content-design/links) under "Writing link text"

Reply:
Thanks!